op-chain-ops/genesis/genesis.go

```diff
@@ -74,16 +74,21 @@ func NewL2Genesis(config *DeployConfig, block *types.Block) (*core.Genesis, erro
 		difficulty = newHexBig(0)
 	}
 
+	extraData := config.L2GenesisBlockExtraData
+	if extraData == nil {
```

Comment:
Should this check for `len(extraData) == 0`?

Reply:
Chose this way because I wondered if there might be a use case where someone would want to genuinely set extradata to 0 bytes.  I think if someone explicitly has empty bytes for extradata in their json file they are probably not wanting the default?

Reply:
Ah ok this totally makes sense